docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -187,21 +327,252 @@ To process the received command on the server side:
+
+   ```
+   mc_playback_action_e playback_action = MC_PLAYBACK_ACTION_PLAY;
+   char *request_id = NULL;
+
+   ret = mc_client_send_playback_action_cmd(g_client_h, server_name, playback_action, &request_id);
+   ```
+
+   If you want to define your own commands to send to the server, use the `mc_client_send_custom_cmd()`.
+   The request_id will be passed to the `mc_client_cmd_reply_recieved_cb()`.
+
+4. Destroy the media controller client handle using the `mc_client_destroy()`, when media controller client handle is no longer needed:
+
+   ```
+   mc_client_destroy(g_client_h);
+   ```
+
+To process the received command on the server side:
+
+1. Create the media controller server handle using the `mc_server_create()`:
+
+   ```
+   ret = mc_server_create(&g_server_h);
+   ```
+
+2. Define the callback that is invoked when the server receives the command.
+
+   For example, to define a callback for playback state change commands:
+
+   ```
+   void
+   playback_action_cmd_received_cb(const char* client_name, const char *request_id, mc_playback_action_e action, void *user_data)
+   {
+       dlog_print(DLOG_DEBUG, LOG_TAG, "Client Name: %s, Request Id: %s, Playback action: %d\n", client_name, request_id, action);
+   }
+   ```
+
+3. Register the callback:
+
+   - To register a callback for playback state change commands, use the `mc_server_set_playback_action_cmd_received_cb()`.
+   - To register a callback for playback position change commands, use the `mc_server_set_playback_position_cmd_received_cb()`.
+   - To register a callback for shuffle mode change commands, use the `mc_server_set_shuffle_mode_cmd_received_cb()`.
+   - To register a callback for repeat mode change commands, use the `mc_server_set_repeat_mode_cmd_received_cb()`.
+   - To register a callback for played item, playback state and playback position change commands in playlist, use the `mc_server_set_playlist_cmd_received_cb()`.
+   - To register a callback for a custom command, use the `mc_server_set_custom_cmd_received_cb()`.
+
+   For example, to register a callback for playback state change commands:
+
+   ```
+   ret = mc_server_set_playback_action_cmd_received_cb(g_mc_server, playback_action_cmd_received_cb, NULL);
+   ```
+
+4. Destroy the media controller server handle using the `mc_server_destroy()`, when media controller server handle is no longer needed:
+
+   ```
+   mc_server_destroy(g_server_h);
+   ```
+
+
+To send the reply of completed command on the server side:
+
+1. Send the reply of completed command using the `mc_server_send_cmd_reply()` with the request id of the command in the third parameter and the result of the command in fourth parameter:
+
+   ```
+   int result_code = 0;
+
+   ret = mc_server_send_cmd_reply(g_server_h, client_name, request_id, result_code, NULL);
+   ```
+
+##### To receive the reply of completed command on the client side:
```

Comment:
Hello, @rishu-samsung 
Avoiding HTML tags are recommended in MD files, not it is not forbidden to use HTML tags.
In case of the efficiency, or some functions not supported by MD files, we can use it.

In this case, to remove HTML tags,
simple sentence are unnaturally changed into title form.

Overuse of the title form is also not recommended,
so for this line and line 543, I think it is better to use HTML tags.

Please let me know your opinion.